README.md

```diff
@@ -1,9 +1,25 @@
+## Generating pre V3.0.0 folder structure for backward compatibility
+In versions >= 3.0.0, the folder structure of FreeRTOS-Plus-TCP has changed and the files have been broken down into smaller logically separated modules. Additionally, the source files have been moved to `source` directory. This change makes existing projects non-compilable and requires modification of project(s) to include the modified source files and include the modified directories.
+
+In case project modification is difficult and/or unwarrented, a script is provided to generate the folder structure similar to [this](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/tree/f118c8415b4373e3d6e6dbd2d5a116f7eaf27b63).
```

Comment:
I agree with @gitwallit's changes.
Beside offering a "compatibility script", I wonder if we should provide some simple examples of how to change a make or a cmake project.
For me it is important to be able to compile my projects both with the earlier as well as with the newer releases of FreeRTOS+TCP.
I made it automatic: "if source/FreeRTOS_IP.c exists, then include files source/*.c, otherwise use the old set of file names".